build-cmakelibs.sh

```diff
@@ -88,9 +102,7 @@ git clone --depth 1 -b release-2.6.3 https://github.com/libsdl-org/SDL_mixer.git
 git clone --depth 1 -b release-2.6.3 https://github.com/libsdl-org/SDL_image.git || { exit 1; }
 git clone --depth 1 -b release-2.20.2 https://github.com/libsdl-org/SDL_ttf.git || { exit 1; }
 
-# We need to clone the whole repo and point to the specific hash for now,
-# till a new version is released after this commit
-
+git clone --depth 1 https://github.com/sahlberg/libsmb2.git || { exit 1; } 
```

Comment:
You need to checkout a concrete hash, take a look how others do (like SDL2)

Reply:
First of all i´m waiting @sahlberg merge this pull request [#](https://github.com/sahlberg/libsmb2/pull/352) then i will open this as soon it´s merged.

Reply:
It has been merged already